main/com.tunein/device.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Tp = require('thingpedia');
+const axios = require('axios');
+const axiosExtensions = require('axios-extensions');
+const url = require('url');
+
+const BASE_URL = 'http://opml.radiotime.com';
+const CONTENT_TYPE = 'application/x-www-form-urlencoded';
+const HTTP_STATUS_OK = 200;
+const SEARCH_PARAM = '/Search.ashx';
+const BROWSE_PARAM = '/Browse.ashx';
+const BROWSE_ITEMS = ['trending', 'music', 'sports', 'talk'];
+
+module.exports = class TuneinRadioDevice extends Tp.BaseDevice {
+    constructor(engine, state, args={}) {
+         super(engine, state);
+         this.uniqueId = 'com.tunein';
+         this.name = 'Tunein';
+         this.description = 'Stream radio through TuneIn API';
```

Comment:
These three are unnecessary, because the base constructor gets this info from the manifest already.